### PR TITLE
Remove Older Ps versions conditions

### PR DESCRIPTION
**Pull request by nenes25**
Hi,

The module claim to be compatible only with Prestashop > 1.7.1.0
But in the code it still remains some conditions for checking older versions.

Regards,

### CODE REVIEW

**Review by MathiasReker — APPROVED** [review]

**Comment by MathiasReker** [comment]
Can we possible have a new release of PrestaShop Cleaner ?

**Comment by PierreRambaud** [comment]
this pr needs to be rebased first :/

**Comment by MathiasReker** [comment]
@PierreRambaud I can do the rebase if needed. I PR's with nice improvements in many native modules. It could be really nice to have those PR's merged :)

**Comment by nenes25** [comment]
Hi,
The rebase is done ;)

File: pscleaner.php [hunk 1]
```diff
@@ -49,25 +47,12 @@ public function __construct()
         $this->description = $this->trans('Check and fix functional integrity constraints and remove default data', array(), 'Modules.Pscleaner.Admin');
         $this->secure_key = Tools::encrypt($this->name);
 	    
-	$this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
+	    $this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
```
Comment (PierreRambaud) [inline pscleaner.php]: Don't mix tab and spaces

Comment (nenes25) [reply pscleaner.php]: Sorry for it, it's fixed now ;)

File: pscleaner.php [hunk 2]
```diff
@@ -38,36 +38,20 @@ public function __construct()
...
         $this->secure_key = Tools::encrypt($this->name);
-	    
-	$this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
+        $this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
     }
 
     protected function getMultiShopValues($key)
     {
-        if (version_compare(_PS_VERSION_, '1.6.0.3', '>=') === true) {
             return Configuration::getMultiShopValues($key);
```
Comment (PierreRambaud) [inline pscleaner.php]:
```suggestion
        return Configuration::getMultiShopValues($key);
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by MathiasReker** [comment]
This PR break the module. I solved the problems here: https://github.com/PrestaShop/pscleaner/pull/38

**Comment by Robin-Fischer-PS** [comment]
Hello @nenes25 ,

Thank you for the PR :)

The module is OK for cleaning Orders and Customers. However, it's NOK for cleaning Catalog : An error occurs when doing this (see video to see the error).

https://drive.google.com/file/d/1p9JXyfXmYb1p7-Kezzxu8DCxXBNlSZou/view

**Comment by nenes25** [comment]
Hi @Robin-Fischer-PS ,

Thanks for the feedback.
My bad on this PR (which was a quick one ) i only remove the condition from the file without testing.

After reviewing in details i see that i misunderstand the way the function version_compare works.
These tables should not being included if  PS version if > 1.7
This is fixed in the last commit

Regards,

**Comment by PierreRambaud** [comment]
Thanks @nenes25

**Pull request merged**